frontend/src/renderer/App.tsx

```diff
@@ -41,12 +93,11 @@ export default function Hello() {
                     <Router>
                         <HeaderResponsive links={randomLinks}/>
                         <Routes>
-                            <Route path="/" element={<Home />}/>
+                            <Route path="/" element={isConfigValid ? <GettingStarted/> : <Config/>}/>
```

Comment:
I am not sure if this will work, if the API returns `JSON` instead of the `true` and false which is wanted in the JSX condition.